Earthfile

```diff
@@ -55,7 +55,17 @@ all-arm:
       BUILD --platform=linux/arm64 +trivy-scan --MODEL=rpi64
       BUILD --platform=linux/arm64 +grype-scan --MODEL=rpi64
   END
-  BUILD +arm-image --MODEL=rpi64
+  
+  IF [[ "$FLAVOR" = "ubuntu-20-lts-arm-nvidia-jetson-agx-orin" ]]
```

Comment:
do we expect other flavors for this boards or ubuntu would be the only one?

Reply:
can't tell, probably this one will be the only one for a long time.. until nvidia releases another version or we have some reques